src/ocean/util/app/ext/UnixSocketExt.d

```diff
@@ -109,10 +109,13 @@ public class UnixSocketExt : IApplicationExtension, IConfigExtExtension
     public void handle ( cstring command, cstring args,
                          void delegate (cstring) send_response )
     {
+        scope pred = (cstring v) { return v.length; };
```

Comment:
I mean that the definition of the `pred` doesn't carry any info of where and for what it will be used (it's not declared even in the same scope). Even moving it to just above the filter will help the reader see that this is the predicate for the `filter`, but I don't see this as a big deal, more a nit-pick, I'm fine to leave it as is, if you insist.